spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/src/main/java/com/example/Trader.java

```diff
@@ -58,9 +58,6 @@ public class Trader {
 	@Column(spannerType = TypeCode.JSON)
 	private TraderDetails details;
```

Comment:
What data type to put for a dynamic JSON which has variable keys and how to handle it?